docs/docs/guides/formatting-your-fields.mdx

```diff
@@ -204,8 +204,46 @@ models:
       meta:
         metrics:
           total_revenue:
-          type: sum
-          round: 2
+            type: sum
+            round: 2
+```
+
+Check out [this doc](https://docs.lightdash.com/references/metrics#metric-configuration) to see all of the other properties you can customize for metrics.
+
+## Compacting big numbers
+
+### To compact your dimensions, you just need to add `compact` to their properties
+
+Like this:
+
+```yaml
+version: 2
+models:
+- name: sales
+  columns:
+    - name: revenue
+      meta:
+        dimension:
+          compact: millions
+```
+
```

Comment:
Can you add an example of what the output would look like here? 

E.g. a screenshot with a compact number